csrc/device_lower/pass/index.cpp

```diff
@@ -2306,9 +2306,22 @@ void IndexLowering::handle(const PadOp* pad) {
 
   pred = GpuLower::current()->commonScalarMap().hoistScalar(pred, for_loops_);
 
+#if 0
+  const auto prev_scope = active_scope_;
```

Comment:
remove commented code and open an issue to track this as alternative for when we needed to lift the ternary into an if statement later.